doc/source/api.rst

```diff
@@ -2328,3 +2565,77 @@ Scalar introspection
     api.types.is_re
     api.types.is_re_compilable
     api.types.is_scalar
+
+
+.. This is to prevent warnings in the doc build. We don't want to encourage
+.. these methods.
+
+.. toctree::
+   :hidden:
+
+   generated/pandas.DataFrame.as_blocks
+   generated/pandas.DataFrame.bfill
+   generated/pandas.DataFrame.blocks
+   generated/pandas.DataFrame.columns
+   generated/pandas.DataFrame.consolidate
+   generated/pandas.DataFrame.ffill
+   generated/pandas.DataFrame.get_value
+   generated/pandas.DataFrame.index
+   generated/pandas.DataFrame.ix
+   generated/pandas.DataFrame.set_value
+   generated/pandas.DataFrame.sortlevel
+
+   generated/pandas.Index.data.rst
+   generated/pandas.Index.asi8.rst
+   generated/pandas.Index.flags
+   generated/pandas.Index.isnull
+   generated/pandas.Index.names
+   generated/pandas.Index.nlevels
+   generated/pandas.Index.notnull
+   generated/pandas.Index.reshape
+   generated/pandas.Index.sort
+   generated/pandas.Index.sortlevel
+
+   generated/pandas.Index.str
+
+   generated/pandas.Panel.as_blocks.rst
+   generated/pandas.Panel.blocks.rst
+   generated/pandas.Panel.consolidate.rst
+   generated/pandas.Panel.fromDict.rst
+   generated/pandas.Panel.isnull.rst
+   generated/pandas.Panel.ix.rst
+   generated/pandas.Panel.notnull.rst
+   generated/pandas.Series.as_blocks
+   generated/pandas.Series.asobject.rst
+   generated/pandas.Series.bfill
+   generated/pandas.Series.blocks
+   generated/pandas.Series.consolidate.rst
+   generated/pandas.Series.ffill
+   generated/pandas.Series.get_value
+   generated/pandas.Series.index
+   generated/pandas.Series.isnull.rst
+   generated/pandas.Series.ix
+   generated/pandas.Series.notnull.rst
+   generated/pandas.Series.set_value
+
+   generated/pandas.Series.cat
+   generated/pandas.Series.dt
+   generated/pandas.Series.str
+
+   generated/pandas.Timestamp.offset
+   generated/pandas.Timestamp.to_datetime
+
+   generated/pandas.Categorical.base.rst
+   generated/pandas.Categorical.itemsize.rst
+   generated/pandas.Categorical.labels.rst
+   generated/pandas.Categorical.nbytes.rst
+   generated/pandas.Categorical.ndim.rst
+   generated/pandas.Categorical.shape.rst
+   generated/pandas.Categorical.size.rst
+   generated/pandas.Categorical.T.rst
```

Comment:
For those I think we should do Categorical also without including all methods by default (as it was now). So those above don't get docstring pages.